frontend/lib/src/components/widgets/DataFrame/hooks/useDataExporter.ts

```diff
@@ -194,6 +194,7 @@ function useDataExporter(
           enforceDownloadInNewTab,
           url,
           filename: suggestedName,
+          setDownloadAttribute: true,
```

Comment:
Default has been `true` so far, so no functional change here.

Reply:
Can you add a comment here that specifies why it's safe to have this always set in this case